Registry team access bulk (#1047)
wandb/docs: content/guides/models/registry/create_registry.md

weight: 2
---

> A custom registry offers flexibility and control over the artifact types that you can use, the visibility of the registry to other teams, and more.

Reviewer: Registry was previously capitalized most of the time
```suggestion
A custom registry offers flexibility and control over the artifact types that you can use, allows you to restrict the registry's visibility, and more.
```

Author: Ah. It should only be capitalized if its a proper noun... 

I  would "App" to try to help distinguish proper vs common noun usage of "registry".  But since removed it...

E.g. "Registry App" vs "a registry ..."

{{% pageinfo color="info" %}}
See the summary table in [Registry types]({{< relref "registry_types.md#summary" >}}) for a complete comparison of core and custom registries.
{{% /pageinfo %}}

## Create a custom registry

> The following procedure describes how to create a custom registry:

Reviewer: ```suggestion
To create a custom registry:
```

1. Navigate to the **Registry** App at https://wandb.ai/registry/.
2. Within **Custom registry**, click on the **Create registry** button.
3. Provide a name for your registry in the **Name** field.
4. Optionally provide a description about the registry.
5. Select who can view the registry from the **Registry visibility** dropdown. See [Registry visibility types]({{< relref "./configure_registry.md#registry-visibility-types" >}}) for more information on registry visibility options.
6. Select either **All types** or **Specify types** from the **Accepted artifacts type** dropdown.
7. (If you select **Specify types**) Add one or more artifact types that your registry accepts.
8. Click on the **Create registry** button. 

{{% alert %}}

> An artifact type cannot be removed from a registry once it is saved in the registry's settings.

Reviewer: ```suggestion
An artifact type cannot be removed from a Registry after it is added to the Registry's settings.
```

> You can not remove an artifact type from a registry once you add and save it in the registry's settings.

Reviewer: ```suggestion
An artifact type cannot be removed from a registry.
```

Author: We need to specify the settings, saving bit.

{{% /alert %}}

{{< img src="/images/registry/create_registry.gif" alt="" >}}

> For example, the preceding image shows a custom registry called `Fine_Tuned_Models` that a user is about to create. The registry is **Restricted** to only members that are manually added to the registry.

Reviewer: It might make more sense to have this paragraph right before the image than right after. Then I know what I'm looking at before I look at it!

> For example, the preceding image shows a custom registry called "Fine_Tuned_Models" that a user is about to create. The registry is set to **Restricted** which means that only members that are manually added to the "Fine_Tuned_Models" registry will have access to this registry.

Reviewer: ```suggestion
For example, the preceding image shows a custom registry called `Fine_Tuned_Models` that a user is about to create. The registry is **Restricted** to only members that are manually added to the registry.
```

> ## Registry visibility types

Reviewer: ```suggestion
## Visibility options
```

> The *visibility* of a registry determines who can access that registry. Restricting the visibility of a custom registry helps ensure that only specified members can access that registry.

Reviewer: ```suggestion
The *visibility* of a registry determines who can access that registry. You can restrict the visibility of a custom registry so that only specified members can access that registry.
```

> There are two type registry visibility options for a custom registry: 

Reviewer: ```suggestion
This table summamrizes the possible visibility options for a custom registry: 
```

> There are two type of registry visibility: *Restricted* and *Organization*. A custom registry can have either a Restricted or Organization visibility. Core registries can only have Organization visibility. 

Reviewer: ```suggestion
There are two registry visibility options for a custom registry:
```
The order here is different from in the table. I am not sure we need to state the two types in the sentence.

| Visibility | Description |
| --- | --- | 
| Restricted   | Only invited organization members can access the registry.| 
| Organization | Everyone in the org can access the registry. |

A team administrator or registry administrator can set the visibility of a custom registry.

The user who creates a custom registry with Restricted visibility is added to the registry automatically as its registry admin. 

## Configure the visibility of a custom registry

> A team administrator or registry administrator can assign the visibility of a custom registry during or after the creation of a custom registry. 

Reviewer: ```suggestion
A team administrator or registry administrator can assign the visibility of a custom registry while creating it or afterward.
```

> The user who creates a custom registry with Restricted visibility is the only user in that registry initially. That user is given an administrative role and they can add other users to that registry.  

Reviewer: ```suggestion
The user who creates a custom registry with Restricted visibility is added to the registry automatically as its registry admin. 
```

> ## Assign the visibility of a custom registry

Reviewer: ```suggestion
## Configure the visibility of a custom registry
```

To restrict the visibility of an existing custom registry:

1. Navigate to the **Registry** App at https://wandb.ai/registry/.
2. Select a registry.
3. Click on the gear icon on the upper right hand corner.
4. From the **Registry visibility** dropdown, select the desired registry visibility.

> Continue if you select **Restricted** visibility:

Reviewer: ```suggestion
5. If you select **Restricted** visibility:
```

<!-- To do: Add updated process of adding a team -->

> 5. Add members of your organization that you want to have access to this registry. Scroll to the **Registry members and roles** section and click on the **Add member** button. 

Reviewer: ```suggestion
  1. Add members of your organization access to this registry in the **Registry members and roles** section. For each member, click **Add member**. 
```
Substeps more accurately reflect the task, and this doesn't artificially break the HTML list up into two in the output.

> 6. Within the **Member** field, add the email or username of the member you want to add.

Reviewer: ```suggestion
  2. In the **Member** field, enter the member's email or username.
```

> 7. Click **Add new member**.

Reviewer: ```suggestion
  3. Click **Add new member**.
```
